scopesim/version.py

```diff
@@ -1,6 +1,19 @@
-version = '0.5.0'
-date    = '2022-04-22 22:00:00 GMT'
+version = '0.5.2'
```

Comment:
The PR mentions 0.5.3; should that be included here already or do you update that after the merge?

Reply:
I'll update that just before I push to Pypi